js/services/util/schema-service.ts

```
@@ -166,15 +166,15 @@ class SchemaService {
         let mainStoreName = this._getMainStoreNameSeed(walletAddress)
 
         return Global.orbitDb.docstore(mainStoreName, {
-            indexBy:"name",
+            indexBy: "_id",
```

I want to leave this name for now until I figure out why old databases aren't opening properly. This would just hide that error instead.